test/runtests.py

```diff
@@ -93,10 +91,10 @@
 tags = set(args.tag or [])
 not_tags = set(args.not_tag or []).union(['fail', 'exclude_' + arch])
 
-if arch_alias:
-    not_tags.add('exclude_' + arch_alias)
-if flavor_alias:
-    not_tags.add('exclude_' + flavor_alias)
+if arch:
+    not_tags.add('exclude_' + arch)
```

Comment:
This dups with line 92 and can be removed. Also for consistency, could you please also remove next `if flavor...` line and add `'exclude_' + flavor` to line 92 union list?

Thanks for also updating this python script!